src/main/jbake/content/servlets.adoc

```diff
@@ -1,19 +1,19 @@
 type=page
 status=published
-title=Java Servlet Technology
+title=Jakarta Servlet Technology
 next=servlets001.html
 prev=jsf-ws007.html
 ~~~~~~
-= Java Servlet Technology
+= Jakarta Servlet Technology
 
 
 [[BNAFD]][[java-servlet-technology]]
 
-18 Java Servlet Technology
---------------------------
+18 Jakarta Servlet Technology
+-----------------------------
```

Comment:
There's lots more underlines that need to be fixed.  I didn't mark them all
but I assume you can find them.